docs/ops/deployment/kubernetes.md

```diff
@@ -143,8 +176,10 @@ data:
     blob.server.port: 6124
     jobmanager.rpc.port: 6123
     taskmanager.rpc.port: 6122
-    jobmanager.memory.process.size: 1472m
-    taskmanager.memory.process.size: 1024m
+    queryable-state.server.ports: 6125
+    jobmanager.memory.process.size: 1600m
+    taskmanager.memory.process.size: 1728m
+    parallelism.default: 2
```

Comment:
Bit torn on this one, but IMO we should ensure that the example definitions work properly even if there is only 1 TM.
Right now this could be problematic since parallelism = 2 but numberOfTaskSlots = 1
(so either reduce parallelism or increase slots)